frontline-functions-quickstart/assets/index.html

```diff
@@ -0,0 +1,110 @@
+            </path>
+          </svg>
+          Live
+        </aside>
+      </nav>
+    </header>
+  </div>
+  <main>
+    <div class="content">
+      <div id="success-header">
+        <img src="https://twilio-labs.github.io/function-templates/static/v1/success.svg" />
+        <div>  
+          <h1>Great news!</h1>
+          <h2>This Frontline Intergration Service using functions has been deployed successully!</h2>
+        </div>
+      </div>
+      <section>
+        <h2>What is this?</h2>
+        <p>
+          This service provides a back-end for your Twilio Frontline Instance.
+        </p>
+      </section>
+      <section>
+        <h2>Next steps:</h2>
+        <p>Configure the callbacks in <a href="https://www.twilio.com/console/frontline">your Frontline instance:</a></p>
+        <form>
+          <ul>
+            <li>
+              <label for="callbacks-crm">CRM Callback</label>
+              <input type="text" id="callbacks-crm" class="function-root" readonly=true value="/callbacks/crm">
+            </li>
+            <li>
+              <label for="callbacks-outgoing-conversation">Outgoing Conversations Proxy</label>
+              <input type="text" id="callbacks-outgoing-conversation" class="function-root" readonly=true
+                value="/callbacks/outgoing-conversation">
+            </li>
+            <li>
+              <label for="callbacks-templates">Templates</label>
+              <input type="text" id="callbacks-templates" class="function-root" readonly=true
+                value="/callbacks/templates">
+            </li>
+            <li>
+              <label for="callbacks-routing">Routing</label>
+              <input type="text" id="callbacks-routing" class="function-root" readonly=true value="/callbacks/routing">
+            </li>
+            <li>
+              <label for="callbacks-twilio-conversations">Converstations Webhooks</label>
+              <input type="text" id="callbacks-twilio-conversations" class="function-root" readonly=true
+                value="/callbacks/twilio-conversations">
+            </li>
+          </ul>
+        </form>
+
+        <br>
+          🔐 All functions exposed by this service are protected by Twilio Webhook authorization tokens, so you won't be
+          able to make requests to them from another app like Postman or CURL.
+        </p>
+        <p><a href="https://www.twilio.com/console/functions/overview/services">📝 Edit in Twilio Functions</a></p>
```

Comment:
This is redundant as the `<!-- APP_INFO_V2 -->` will automatically insert the "Edit" button